scripts/update-extension.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+source .env
+RELEASE_NAMESPACE="${RELEASE_NAMESPACE:-arc-osm-system}"
+EXTENSION_NAME="${EXTENSION_NAME:-osm}"
+API_VERSION="${API_VERSION:-2020-07-01-preview}"
+EXTENSION_SETTINGS=$1
+
+export RESOURCEID=subscriptions/$SUBSCRIPTION/resourceGroups/$RESOURCEGROUP/providers/Microsoft.Kubernetes/connectedClusters/$CLUSTERNAME
+
+echo $RESOURCEID
```

Comment:
did you mean to have this echo in the script?

Reply:
I left it in since it can be useful for debugging (these are dev only scripts). We have it in the add-extension.sh script as well.